docs/core/tools/csproj.md

```diff
@@ -56,7 +56,7 @@ In order to get around this error, you can either remove the explicit `Compile`
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
 </PropertyGroup>
 ```
-Setting this property to `false` will override implicit inclusion and the behavior will revert back to the previous SDKs where you had to specify the default globs in your project. 
+Setting this property to `false` will disable implicit inclusion and the behavior will revert back to that of the previous SDKs where you had to specify the default globs in your project. 
```

Comment:
"to that of" sounds weird.

Reply:
it's actually more correct than the original. It's the behavior that reverts, not the SDK.

Reply:
Would this be clearer?

> Setting this property to `false` will disable implicit inclusion, restoring the behavior of previous SDKs where you had to specify the default globs in your project.

Reply:
yes!

Reply:
Done.